build.sh

```diff
@@ -34,7 +34,7 @@ list_platforms() {
 }
 
 do_build_native() {
-    if [[ $( dpkg --print-architecture | head -1 ) != "${PLATFORM##*.}" ]]; then
+    if [[ -f $( which dpkg ) && $( dpkg --print-architecture | head -1 ) != "${PLATFORM##*.}" ]]; then
```

Comment:
this isn't doing what you wanted it to do :) still not giving an error on non-Debian :)
try this (untested):
```suggestion
    if [[ ! -f $( which dpkg ) || $( dpkg --print-architecture | head -1 ) != "${PLATFORM##*.}" ]]; then
```